packages/shortest/src/cli/init/index.ts

```diff
@@ -20,12 +20,12 @@ export default async function main() {
       packageJson?.devDependencies?.["@antiwork/shortest"]
     ) {
       console.log(pc.green("✔ Package already installed"));
-      return;
```

Comment:
The current behaviour is intentional. If the package is already present, we can assume that some sort of setup has already been done. Otherwise, trying to do an incremental install could trigger various edge cases that are not worth supporting at this time. 

See https://github.com/anti-work/shortest/pull/297/files#r1935142270

Reply:
noted, thanks for pointing out